### ISSUE TITLE
2W at night

### ISSUE THREAD

**Issue by dsteinkopf** [lumapu/ahoy#575]
### Platform

ESP8266

### Model name

Wemos D1 mini

### nRF24L01+ Module

nRF24L01+ plus

### Antenna

circuit board

### Power Stabilization

~100uF Elko

### Connection diagram

"Normal" - as described in the readme + default config.

### Connection picture

- [ ] I will attach/upload an Image of my wiring

### Version

0.5.66

### Github Hash

f8fe044

### Build & Flash Method

ESP Tools (flash)

### Desktop

Mac OS

### Setup

Does this matter for the described problem? I'll add this here if needed.

### Error description

Last night I had the issue that my inverter seemed to constantly produce 2.2W. I assume, this was because the night came "too quickly" and the inverter stopped responding before sending "0W" at least once. This also was an issue here: henne49/dbus-opendtu#21. But we'd consider this also a bug in Ahoy:

In `http://ahoy/api/live` the `ts_last_success` is correctly showing that the values are old. One could discuss if is acceptable that the old values (2.2W in this case) are sent out through the hole night. - In my eyes this is ok as we have the timestamp.

The problem I see is that the **latest value which is sent via MQTT is also 2.2W**. So e.g. HomeAssistant constantly shows this value - which is wrong.

**Comment by lumapu** [comment]
please check the latest development firmware which includes options to zero values by different triggers. They can be set inside the settings window inside the MQTT region.

Dev build download:
https://ahoydtu.de/web_install then click the link for development version

**Comment by TB-79** [comment]
I do Not use mqtt, Just mystrom and have this too  over night
![Screenshot_2023-01-10-21-33-41-26_465c34575edd84c613cd0068e946753f](https://user-images.githubusercontent.com/121585700/211758460-e6aec27f-071a-4826-94b3-3a0f95f6a0da.jpg)

**Comment by stefan123t** [comment]
@TB-79 I consider your problem with mystrom a measurement issue of the mystrom device you are plugging between the HM-Inverter and your wall-wart. This is measuring the AC side of the Inverter and therefor it may account some power detected by small power factor changes in the grid.
Your issue is on the grid side of things and in how your mystrom Wifi switch works. We cannot do anything about that IMHO.  

The problem that we are focusing on here and that @dsteinkopf had raised is the messages via MQTT are sent with a retained flag and therefor could not be zeroed during disabled night communication (the inverter will not be able to send any new data after sunset) and/or the yield daily can now also be reset at midnight (in case you are running a battery to power your inverter).
These issues are both fixed in latest development build.

**Comment by dsteinkopf** [comment]
I've now updated to dev version `d8e255d :: 0.5.70` and will see if this happens again.

BTW. I was not able to do an OTA. For debugging I connected a serial cable and saw it was an OOM exception. So I uploaded the firmware manually via serial cable. All settings were lost. Is that expected?

Also I was not able to upload the previously downloaded `ahoy_setup.json`. Console show the exception below. Is that expected?

```
[09:30:37]--------------- CUT HERE FOR EXCEPTION DECODER ---------------
[09:30:37]
[09:30:37]Exception (28):
[09:30:37]epc1=0x4000df2f epc2=0x00000000 epc3=0x00000000 excvaddr=0x00000033 depc=0x00000000
[09:30:37]
[09:30:37]>>>stack>>>
[09:30:37]
[09:30:37]ctx: sys
[09:30:37]sp: 3fffea00 end: 3fffffb0 offset: 0190
[09:30:37]3fffeb90:  40249c05 3fff3014 4026cf40 401005c8  
[09:30:38]3fffeba0:  40247909 00010083 0000010f 40260d50  
[09:30:38]3fffebb0:  00000033 00000010 402481fd 3ffee3c0  
[09:30:38]3fffebc0:  3fff3014 00000000 3ffe9b19 3fff3014  
[09:30:38]3fffebd0:  3ffe9b19 3fffc200 00000022 000000ff  
[09:30:38]3fffebe0:  3fff6025 3fff2d6c 00000007 3ffeeb04  
[09:30:38]3fffebf0:  3ffe9b19 3ffe9b19 00000033 4024a0ab  
[09:30:38]3fffec00:  3ffeeab8 40249c7c 4024c2f0 3ffeeab8  
[09:30:38]3fffec10:  402497fc 3fffed98 80808080 3fff2152  
[09:30:38]3fffec20:  00000001 00000001 3fff12a2 00000030  
[09:30:38]3fffec30:  40249cb7 3ffeebb0 3ffeebb0 40100280  
[09:30:38]3fffec40:  4024bc30 00000000 4025f50f 00000000  
[09:30:38]3fffec50:  3ffee444 4024bbf4 3ffee3c0 00000012  
[09:30:38]3fffec60:  40249c6c 40249c40 3ffee444 00000012  
[09:30:38]3fffec70:  40249c0b 3ffee3c0 00000012 40265094  
[09:30:38]3fffec80:  4024a1bc 3ffee3c0 00000000 4024c2f0  
[09:30:38]3fffec90:  40249be9 3ffee53c 00000002 00000000  
[09:30:38]3fffeca0:  4024c0a5 0000010f 00000000 0000003e  
[09:30:38]3fffecb0:  3fff12a2 0000010f 3ffee5dc 4024bef9  
[09:30:38]3fffecc0:  4024bf04 3fff2152 00000000 00000001  
[09:30:38]3fffecd0:  402656c0 3fff2152 00000000 00000000  
[09:30:38]3fffece0:  3fff2d6c 4021963e 3fff12c2 40219567  
[09:30:38]3fffecf0:  77706970 006e616c 3ffffe00 40241395  
[09:30:38]3fffed00:  402643a1 00000001 00000001 402392a0  
[09:30:38]3fffed10:  49524537 36485453 4c49584e 34345073  
[09:30:38]3fffed20:  72674c6e 76587559 5a705972 6b345536  
[09:30:38]3fffed30:  4737567a 4453556d 6b694175 6e653336  
[09:30:38]3fffed40:  67516a58 52526f67 376a5873 3f006957  
[09:30:38]3fffed50:  88da7400 00609abe 3fff2154 00000030  
[09:30:38]3fffed60:  77706970 006e616c 3ffed7f0 00000000  
[09:30:38]3fffed70:  40104f25 3ffee2a0 3ffed9d0 00040000  
[09:30:38]3fffed80:  49524537 36485453 4c49584e 34345073  
[09:30:38]3fffed90:  72674c6e 76587559 5a705972 6b345536  
[09:30:38]3fffeda0:  4737567a 4453556d 6b694175 6e653336  
[09:30:38]3fffedb0:  67516a58 52526f67 376a5873 3f006957  
[09:30:38]3fffedc0:  00000000 2c9f0300 40000581 00000002  
[09:30:38]3fffedd0:  00000000 3fff2150 00000001 00000004  
[09:30:38]3fffede0:  40218ae1 00000030 00000008 3fff2154  
[09:30:38]3fffedf0:  00000001 3fff2150 3ffefaf0 40219694  
[09:30:38]3fffee00:  3fffeef2 0000009a 00000060 40217df2  
[09:30:38]3fffee10:  3fffeee0 3fff7b4f 00000001 3fff7b44  
[09:30:38]3fffee20:  3fff5c44 00000001 3fff2154 00000030  
[09:30:38]3fffee30:  00000000 3fff2bec 3fff7f94 3fff2be8  
[09:30:38]3fffee40:  3fff7f94 00000020 3fff65a4 40100c6a  
[09:30:38]3fffee50:  3fff2bf4 00000000 00000000 401005ec  
[09:30:38]3fffee60:  3fff2d6c ffffffcc 3fff7f94 402413bc  
[09:30:38]3fffee70:  00000001 3fff5c44 3ffefaf0 40217eb4  
[09:30:38]3fffee80:  3fff7f94 3fff2c0c 3ffefaf0 4021805b  
[09:30:38]3fffee90:  00000000 4bc6a7f0 00016065 401005c8  
[09:30:38]3fffeea0:  0000012c 00000001 3fffeee0 40227ab2  
[09:30:38]3fffeeb0:  3fffeee0 3fff7b48 3fff7b44 40218098  
[09:30:38]3fffeec0:  3fffdc10 3fff2048 00000001 4022c8d4  
[09:30:38]3fffeed0:  00000001 3fff5c44 3fff7b44 40218ae4  
[09:30:38]3fffeee0:  77706970 006e616c 07000000 be88da74  
[09:30:38]3fffeef0:  0000609a 00000008 3fff8bac 4021a328  
[09:30:38]3fffef00:  0000001e 3fff7b44 3fff5c2c 40218e7c  
[09:30:38]3fffef10:  3fff7b44 40101dff 00000000 3ffedec4  
[09:30:38]3fffef20:  3ffed9d0 40101bd0 00000000 00000000  
[09:30:38]3fffef30:  40101c05 00000000 00000002 3ffef498  
[09:30:38]3fffef40:  3fffdc10 00000000 3fff7b44 40266cd4  
[09:30:38]3fffef50:  40000f49 3fffdab0 3fffdab0 40000f49  
[09:30:38]3fffef60:  40000e19 0007ac91 00000000 00000005  
[09:30:38]3fffef70:  3fffefc0 aa55aa55 000000ed 401055f1  
[09:30:38]3fffef80:  401055f7 00000000 00000005 40100c44  
[09:30:38]3fffef90:  4010000d 64616f6c 0007ac91 401000ac  
[09:30:38]3fffefa0:  40253304 3fffef3c 402532b5 3ffffe68  
[09:30:38]3fffefb0:  3fffffc0 00000000 00000000 feefeffe  
[09:30:38]3fffefc0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3fffefd0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3fffefe0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3fffeff0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff000:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff010:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff020:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff030:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff040:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff050:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff060:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff070:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff080:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff090:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff0a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff0b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff0c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff0d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff0e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff0f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff100:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff110:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff120:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff130:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff140:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff150:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff160:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff170:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff180:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff190:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff1a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff1b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff1c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff1d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff1e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff1f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff200:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff210:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff220:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff230:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff240:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff250:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff260:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff270:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff280:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff290:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff2a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff2b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff2c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff2d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff2e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff2f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff300:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff310:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff320:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff330:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff340:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff350:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff360:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff370:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff380:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff390:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff3a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff3b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff3c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff3d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff3e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff3f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff400:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff410:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff420:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff430:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff440:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff450:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff460:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff470:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff480:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff490:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff4a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff4b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff4c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff4d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff4e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff4f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff500:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff510:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff520:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff530:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff540:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff550:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff560:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff570:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff580:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff590:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff5a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff5b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff5c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff5d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff5e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff5f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff600:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff610:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff620:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff630:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff640:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff650:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff660:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff670:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff680:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff690:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff6a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff6b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff6c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff6d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff6e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff6f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff700:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff710:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff720:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff730:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff740:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff750:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff760:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff770:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff780:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff790:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff7a0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff7b0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff7c0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff7d0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff7e0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff7f0:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff800:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff810:  feefeffe feefeffe feefeffe feefeffe  
[09:30:38]3ffff820:  4000b570 feefeffe feefeffe feefeffe  
[09:30:38]3ffff830:  00000000 00000000 00000000 00000000  
[09:30:38]3ffff840:  00000000 00000000 00000000 00000000  
[09:30:38]3ffff850:  00000000 00000000 00000000 00000000  
[09:30:38]3ffff860:  00000000 00000000 00000000 00000000  
[09:30:38]3ffff870:  1b866ce1 915bb950 fc0e574d 2e377469  
[09:30:38]3ffff880:  71947d1b 8f111401 73dfb778 b21bb5f7  
[09:30:38]3ffff890:  0f565386 bfe04cb1 bca1a05e 90458348  
[09:30:38]3ffff8a0:  4000b5f9 00000008 00000038 3ffff980  
[09:30:38]3ffff8b0:  00000008 3ffff980 22336aa7 74ca9f77  
[09:30:38]3ffff8c0:  4038e99b a01c766d 2bffc121 64ed1cc6  
[09:30:38]3ffff8d0:  8dda5426 75b9431d 0a2454b0 9bb3c637  
[09:30:38]3ffff8e0:  f1ab5c0b 8c0822ab 5950acbd b50f337a  
[09:30:38]3ffff8f0:  e3514b79 a15f81e9 1ed73d2d 83cba813  
[09:30:38]3ffff900:  84d5ff9f beb2e68d 00e3a32f ec77b403  
[09:30:38]3ffff910:  d817ad2a 8ffedd7f dfe21b79 ff91eff2  
[09:30:38]3ffff920:  7c4a22ad b82c4472 40a37d42 c6b25200  
[09:30:39]3ffff930:  4000b62e 38f24c93 68e73d64 7b1a62a4  
[09:30:39]3ffff940:  4000b6c4 3ffff980 3ffffb10 ffffffff  
[09:30:39]3ffff950:  4000b72e 3ffff960 00000008 f94fe682  
[09:30:39]3ffff960:  00000000 00000000 4000a333 3ffffa88  
[09:30:39]3ffff970:  3ffffb10 3ffffa68 3fff6694 ca84d3d3  
[09:30:39]3ffff980:  00000000 00000000 00000000 00000000  
[09:30:39]3ffff990:  00000000 00000000 0000001f 40100280  
[09:30:39]3ffff9a0:  00000000 00000000 3fffc228 40105d89  
[09:30:39]3ffff9b0:  4000050c 00000000 00000000 00000000  
[09:30:39]3ffff9c0:  4023483e 00000030 00000000 ffffffff  
[09:30:39]3ffff9d0:  4022fd72 3ffe8368 3ffffc20 3ffffbb3  
[09:30:39]3ffff9e0:  00000000 00000000 0000001f 40100280  
[09:30:39]3ffff9f0:  000000b0 00000006 3fffc228 40105d89  
[09:30:39]3ffffa00:  4000050c 00000000 3ffe8368 3ffe8368  
[09:30:39]3ffffa10:  4023483e 00000030 00000000 ffffffff  
[09:30:39]3ffffa20:  4022fd72 00000000 3ffffc70 000000d0  
[09:30:39]3ffffa30:  00000091 00000001 4010543d 3ffee2e8  
[09:30:39]3ffffa40:  3ffee270 00000006 3ffffc03 3ffffc70  
[09:30:39]3ffffa50:  00000000 00000000 3ffe8368 00000030  
[09:30:39]3ffffa60:  40105022 3ffee2e8 3ffedb38 40102258  
[09:30:39]3ffffa70:  40102de3 40104971 3ffee2e8 40100280  
[09:30:39]3ffffa80:  00007fff 04b14169 3ffee444 40102fc0  
[09:30:39]3ffffa90:  40104f25 3ffee2e8 3ffed890 00040000  
[09:30:39]3ffffaa0:  53002200 04b14169 00000002 40102258  
[09:30:39]3ffffab0:  401051fd 00080000 00000002 00000001  
[09:30:39]3ffffac0:  401034ca 401046b2 3ffee2e8 40102258  
[09:30:39]3ffffad0:  40104b23 00000000 3ffee444 40102fc0  
[09:30:39]3ffffae0:  00000000 2c9f0300 4000050c 3fffc278  
[09:30:39]3ffffaf0:  4010319c 3fffc200 00000022 40105d89  
[09:30:39]3ffffb00:  40100c97 00000030 00000010 3fff279c  
[09:30:39]3ffffb10:  3fff279c fffffff4 3fff5ed4 40238e0e  
[09:30:39]3ffffb20:  3fff27d4 3fff5ed4 00000000 40239252  
[09:30:39]3ffffb30:  3fff2ab8 00000000 00000002 00000020  
[09:30:39]3ffffb40:  00000014 3fff867c 00000046 3fff2ab8  
[09:30:39]3ffffb50:  3fff279c 3fff27d4 3fff2aac 4023f004  
[09:30:39]3ffffb60:  00000000 00000001 00000005 40102258  
[09:30:39]3ffffb70:  3fff5ed4 3ffee2e8 3ffedb38 00000002  
[09:30:39]3ffffb80:  00000be9 00000be9 3ffe85e0 3fff5ed4  
[09:30:39]3ffffb90:  3fff5ed4 3fff5fd0 3fff279c 4023f0d8  
[09:30:39]3ffffba0:  00000000 00000d1b 3ffe85e0 40100a8b  
[09:30:39]3ffffbb0:  3fff5f18 3fff279c 3fff5fd0 402408d0  
[09:30:39]3ffffbc0:  3fff950c 00000020 3fff950c 40100c6a  
[09:30:39]3ffffbd0:  00000014 3fff5fcc 000000ff 00000000  
[09:30:39]3ffffbe0:  00000654 00000654 3ffe85e0 40100a8b  
[09:30:39]3ffffbf0:  3fff5fcc 00000014 00000020 3fff5fd0  
[09:30:39]3ffffc00:  3fff279c 00000020 3fff5ed4 40100c6a  
[09:30:39]3ffffc10:  3fff279c 00000000 00000000 401005ec  
[09:30:39]3ffffc20:  4022b24c 00000020 3fff8a3c 402413bc  
[09:30:39]3ffffc30:  3fff279c 00000000 00000000 4023af46  
[09:30:39]3ffffc40:  4022b24c 3fff5ed4 3fff5fcc 4023c016  
[09:30:39]3ffffc50:  00000673 00000673 3ffe85e0 40100a8b  
[09:30:39]3ffffc60:  3fff5fcc fffffff4 3fff5ed4 3fff5fcc  
[09:30:39]3ffffc70:  40219a20 00000020 3fff5fcc 40100c6a  
[09:30:39]3ffffc80:  3fff5fcc 00000001 0000ea76 401005ec  
[09:30:39]3ffffc90:  00000014 00006008 3fff91a4 402413bc  
[09:30:39]3ffffca0:  0000ea76 0000075b 3fff5fcc 402392f5  
[09:30:39]3ffffcb0:  3fff6520 3fff6514 3fff5fcc 40239e04  
[09:30:39]3ffffcc0:  40219a20 00000001 0000ea76 4023a496  
[09:30:39]3ffffcd0:  0000075b 00000022 3fff3640 40220245  
[09:30:39]3ffffce0:  00004c40 b8c0aed1 3fff5c6c 00000000  
[09:30:39]3ffffcf0:  00000011 3fff2bac 00000020 3ffe9bac  
[09:30:39]3ffffd00:  00000a3b 00000a3b 3ffe85e0 40100a8b  
[09:30:39]3ffffd10:  0128a8c0 00ffffff f828a8c0 3fff92d4  
[09:30:39]3ffffd20:  3fff9430 00000020 3fff7e0c 40100c6a  
[09:30:39]3ffffd30:  4026f388 3fff2a6c 00000000 401005ec  
[09:30:39]3ffffd40:  00000000 00000020 3fff92d4 402413bc  
[09:30:39]3ffffd50:  3ffe9bac 3ffe9bac 3fff279c 402392f5  
[09:30:39]3ffffd60:  f828a8c0 0000003a 000000fa 4023bd32  
[09:30:39]3ffffd70:  3fff9430 3fff279c 3fff7dd4 4023da30  
[09:30:39]3ffffd80:  3fff9430 3fff279c 3fff7dd4 4023e6f4  
[09:30:39]3ffffd90:  0128a8c0 00ffffff 00000000 00000000  
[09:30:39]3ffffda0:  3fff279c 3fff2b0c 00000004 40238e3d  
[09:30:39]3ffffdb0:  3fff279c 00000000 3fff279c 4023b6f1  
[09:30:39]3ffffdc0:  00000a1f 00000a1f 3ffe85e0 40100a8b  
[09:30:39]3ffffdd0:  00000000 3ffea380 00000018 00000000  
[09:30:39]3ffffde0:  4024cb23 4025d05a 00000000 3ffef884  
[09:30:39]3ffffdf0:  3ffefabe 40261256 00000000 3ffef884  
[09:30:39]3ffffe00:  4026cd68 3ffea910 00000000 3ff1fe00  
[09:30:39]3ffffe10:  4025d05f 00000000 00000000 00000000  
[09:30:39]3ffffe20:  40261271 00000043 3fff3014 40100c6a  
[09:30:39]3ffffe30:  4026133d ffffffff 3fff2d6c 401005ec  
[09:30:39]3ffffe40:  40105ecd 01ae04ee 3ffeef80 00000000  
[09:30:39]3ffffe50:  3ffef2f0 3ffeef80 3ffee3c0 00000000  
[09:30:39]3ffffe60:  00000000 01ad8adc 3ffef764 3ffef884  
[09:30:39]3ffffe70:  3ffefabe 3ffefa64 40228974 3fffefa0  
[09:30:39]3ffffe80:  40224899 00000001 3ffffea0 4022489c  
[09:30:39]3ffffe90:  00000000 3ffef884 3ffef764 40207aa2  
[09:30:39]3ffffea0:  00000000 000c000f 00000000 3ffefa64  
[09:30:39]3ffffeb0:  3ffefabb 3ffef830 3ffef764 4022bee8  
[09:30:39]3ffffec0:  3fffdad0 00000000 3ffef764 402072bf  
[09:30:39]3ffffed0:  01010101 01010101 00010101 00000000  
[09:30:39]3ffffee0:  00000000 00000000 00000000 40220000  
[09:30:39]3ffffef0:  3ffefab4 3ffffeda 3ffef760 3fff2364  
[09:30:39]3fffff00:  3fff1160 3fff04a0 3ffef760 3fff2364  
[09:30:39]3fffff10:  3fffdad0 00000000 3ffef760 4020eb5e  
[09:30:39]3fffff20:  00000005 3fff1860 3fff1238 4022c260  
[09:30:39]3fffff30:  3ffef764 3fff1860 3fff1238 40216069  
[09:30:39]3fffff40:  3fff78f4 00000000 40205850 4022bfcc  
[09:30:39]3fffff50:  3fff6d94 00000000 402058f8 3fff2364  
[09:30:39]3fffff60:  007a1200 6d04eeb3 00000000 402296ed  
[09:30:39]3fffff70:  00000000 00000000 3fff3908 0000001b  
[09:30:39]3fffff80:  00000000 00000000 00000001 3fff2364  
[09:30:39]3fffff90:  3fffdad0 00000000 3fff2350 402176f8  
[09:30:39]3fffffa0:  feefeffe feefeffe 3fff2350 40228aa0  
[09:30:39]<<<stack<<<
[09:30:39]
[09:30:39]--------------- CUT HERE FOR EXCEPTION DECODER ---------------
```

**Comment by dsteinkopf** [comment]
Similar (same?) problem during the day after a reboot of Ahoy: 
**Values are sent out as 0**, although Ahoy seems to know that this is wrong as **ts_last_success is also 0**.
So in this case, the values should just not be sent out. Should I open a new issue for this?

**Comment by stefan123t** [comment]
Sending out values with ts_last_success being 0 might qualify as a bug/issue.
@lumapu can you qualify that ?
Regarding the reboots / stack traces I do not know wether we can decode them without the right version / githash ?

**Comment by Duesentrieb5413** [comment]
I have the same issue with Ahoy DTU not resetting the values with no updated data.
I tried the current dev-version 0.5.75 with some remarks:
- Using the option "Reset Values at Communication stop" resulted in reset values all the time (related to this issue). When I deactivated this option, updated values and timestamp were shown as expected: ![2023-01-22 11_45_52-Live – Mozilla Firefox](https://user-images.githubusercontent.com/25255380/213915443-8b473b7c-377c-4801-9e66-bfe3adde4695.png)
- installing via webinstaller did not work - I always got the error "Failed to fetch" (Chrome was started with parameter). Thus flashed via https://github.com/nodemcu/nodemcu-flasher
![2023-01-22 13_23_37-Flash _ AhoyDTU](https://user-images.githubusercontent.com/25255380/213915625-8f693082-d142-487b-8bac-d43131cec5be.png)
- The build was not stable - after some time the website of Ahoy did not show menu/values any more
![2023-01-22 13_11_17-](https://user-images.githubusercontent.com/25255380/213915460-026f2795-5b7a-4f9d-bd23-57bd1633fe9d.png)
Should I open additional issues for my second and third remark?
